test cases/common/262 configure file output format/meson.build

```diff
@@ -0,0 +1,34 @@
+project('configure file output format')
+
+data = configuration_data()
+data.set_quoted('str', 'hello world!', description: '''This is
+a multiline
+description''')
+data.set('int', 42, description: 'ultimate question of life, the universe, and everything')
+data.set('bool', true)
+data.set('false', false)
+
+config_h = configure_file(
+    configuration: data,
+    output_format: 'c',
+    output: 'config.h'
+)
+
+config_nasm = configure_file(
+    configuration: data,
+    output_format: 'nasm',
+    output: 'config.nasm'
+)
+
+config_json = configure_file(
+    configuration: data,
+    output_format: 'json',
+    output: 'config.json'
+)
+
+pymod = import('python')
+py = pymod.find_installation('python3')
```

Comment:
Why not `find_program('python3')`? Note that I *fixed* tons of test cases to use find_program instead, in #10361.